src/test/test_bitcoin.h

```diff
@@ -67,6 +67,7 @@ struct TestingSetup: public BasicTestingSetup {
 
     explicit TestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
     ~TestingSetup();
+    fs::path getPath(const std::string& name);
```

Comment:
I suppose if you move getPath to BasicTestingSetup, you don't have to move `util_tests` and `dbwrapper_tests` to `TestingSetup`?

Reply:
I was using the existing (unused?) `pathTemp` in `TestingSetup` as a root directory, I could move that and the new method to `BasicTestingSetup` if think that would be more appropriate.